### ISSUE TITLE
dispute-distribution/dispute-coordinator: Optimize vote import

### ISSUE THREAD

**Issue by eskimor** [paritytech/polkadot#4404]
As can be seen here

![Screenshot from 2021-11-29 17-57-55](https://user-images.githubusercontent.com/1527017/143910441-c53169b0-f8dc-4a6c-b5e8-01bf8a5eedf5.png)

vote import from dispute distribution is not terribly efficient. It looks like we are only able to import like 15-20 votes per second. One reason this could be, is that we are writing to disk after every import. One way to mitigate this, would be to batch together votes in dispute distribution as much as we can once we wait on dispute-coordinator for further imports. Another potentially easier and better way is to adjust the disk flush technique used in dispute-coordinator.

While optimizing db flushes, almost certainly is a useful optimization, ideally we would add some measurements to see where the time on import is actually spent the most.

# Importance

I think we are fine - senders will retry if the import fails, but: Only successfully imported disputes can be prioritized properly, so the better the import (of new disputes), the better the participation strategy.

**Comment by rphmeier** [comment]
Duplicate of #3437 ?

**Comment by eskimor** [comment]
Yep, good catch!
